src/main/java/frc/lib/tuneables/SendableType.java

```diff
@@ -0,0 +1,16 @@
+package frc.lib.tuneables;
+
+public enum SendableType {
+    NONE(null),
+    LIST("LW Subsystem");
+
+    private String stringType;
+
+    private SendableType(String stringType) {
+        this.stringType = stringType;
+    }
+
+    public String getStringType() {
+        return stringType;
```

Comment:
this.stringType is more elegant

Reply:
Like this?
```java
public String getStringType() {
    return this.stringType;
}
```
It's just a getter, since when do getters using `this` is more elegant? Getters are one of the only methods that will always refer to a member of the class, you can't really confuse it with anything else. I will even argue this makes it less elegant since this is not how getters are used to be written, making it less conventional.

Reply:
idk it makes more sense to me